src/GafferDispatch/Dispatcher.cpp

```diff
@@ -434,16 +438,13 @@ class Dispatcher::Batcher
 			{
 				float frame = task.context()->getFrame();
 				std::vector<float> &frames = batch->frames();
-				if( std::find( frames.begin(), frames.end(), frame ) == frames.end() )
```

Comment:
I was just wondering about that. I don't remember why it was added originally though... so if the tests are still passing...